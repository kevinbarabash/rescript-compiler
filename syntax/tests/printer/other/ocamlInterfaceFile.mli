(* parses ocaml interface file *)
val x : int
