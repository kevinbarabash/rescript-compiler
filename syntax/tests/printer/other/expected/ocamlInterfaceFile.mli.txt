/* parses ocaml interface file */
let x: int
